style_guide.md

```diff
@@ -0,0 +1,115 @@
+### Variables
+
```

Comment:
Could have some examples of the good and bad in this section?

Reply:
Added! Thanks👍